ports/esp32/machine_sdcard.c

```diff
@@ -211,7 +211,7 @@ STATIC mp_obj_t machine_sdcard_make_new(const mp_obj_type_t *type, size_t n_args
                 .gpio_cs = GPIO_NUM_34,
                 .gpio_cd = SDSPI_SLOT_NO_CD,
                 .gpio_wp = SDSPI_SLOT_NO_WP,
-                .dma_channel = 2
+                .dma_channel = 3
```

Comment:
Probably worth using the `SPI_DMA_CH_AUTO` here, it will be available because ESP32-S3 is only built with IDF 4.4.

Reply:
Thank you. Replaced it. https://github.com/micropython/micropython/pull/10490/commits/d6816345e27f844e8ab85f5c92d0ce85af1c7161

Reply:
Thanks, I'm in the process of rebasing this and merging.